source/gui/settingsDialogs.py

```diff
@@ -507,7 +507,7 @@ def onCharHook(self,evt):
 				self.currentCategory.SetFocus()
 		elif listHadFocus and key == wx.WXK_RETURN:

```

Comment:
Should this one also handle `wx.WXK_NUMPAD_ENTER`? (Pressing numpad enter on the categories list)

Reply:
Hmm, your question actually triggered me to do some investigation, and it looks like the [wx bug in question](http://trac.wxwidgets.org/ticket/3725) does not hurt us any more. Therefore this part of the code can also be removed. I'll do some extra checks with older NVDA versions to be 100% sure.